pkg/systemlogmonitor/logwatchers/journald/log_watcher.go

```diff
@@ -146,19 +159,6 @@ func getJournal(cfg types.WatcherConfig) (*sdjournal.Journal, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to create journal client from path %q: %v", path, err)
 	}
-	// Use system uptime if lookback duration is longer than it.
-	// Ideally, we should use monotonic timestamp + boot id in journald. However, it doesn't seem
-	// to work with go-system/journal package.
-	// TODO(random-liu): Use monotonic timestamp + boot id.
-	var info syscall.Sysinfo_t
-	if err := syscall.Sysinfo(&info); err != nil {
-		return nil, fmt.Errorf("failed to get system info: %v", err)
-	}
-	uptime := time.Duration(info.Uptime) * time.Second
-	if lookback > uptime {
-		lookback = uptime
-		glog.Infof("Lookback changed to system uptime: %v", lookback)
-	}
 	// Seek journal client based on the lookback duration.
 	start := time.Now().Add(-lookback)
```

Comment:
Why not use this?

Reply:
Now considering lookback when getting start time.